lib/sles4sap_publiccloud.pm

```diff
@@ -50,6 +50,8 @@ our @EXPORT = qw(
   create_instance_data
   deployment_name
   delete_network_peering
+  get_hana_database_info
```

Comment:
`get_hana_database_info`  does not need to be exported as only used internally in the lib

Reply:
ok good point